apps/src/aichat/views/ChatEventsList.tsx

```diff
@@ -0,0 +1,53 @@
+import React, {useEffect, useRef} from 'react';
+
+import {useAppSelector} from '@cdo/apps/util/reduxHooks';
+
+import {ChatEvent} from '../types';
+
+import ChatEventView from './ChatEventView';
+
+import moduleStyles from './chatWorkspace.module.scss';
+
+interface ChatEventsListProps {
+  events: ChatEvent[];
+  showWaitingAnimation: () => React.ReactNode;
```

Comment:
Not your change, but passing a function instead of a boolean here reads a little weird to me (maybe because previously it was an internal function?). Not necessary for this PR, but maybe we could change this to a `showWaitingAnimation: boolean` and have this component render the appropriate waiting image if true?

Reply:
Oh yeah - I agree a `boolean` would be expected here. I'll update in follow-up!